Code/Framework/AzFramework/AzFramework/Spawnable/Spawnable.h

```diff
@@ -29,7 +30,143 @@ namespace AzFramework
         AZ_CLASS_ALLOCATOR(Spawnable, AZ::SystemAllocator, 0);
         AZ_RTTI(AzFramework::Spawnable, "{855E3021-D305-4845-B284-20C3F7FDF16B}", AZ::Data::AssetData);
 
+        // The order is important for sorting in the SpawnableAssetHandler.
+        enum class EntityAliasType : uint8_t
+        {
+            Original,   //!< The original entity is spawned.
+            Disabled,   //!< No entity will be spawned.
+            Replace,    //!< The entity alias is spawned instead of the original.
+            Additional, //!< The original entity is spawned as well as the alias. The alias will get a new entity id.
+            Merge       //!< The original entity is spawned and the components of the alias are added. The caller is responsible for
+                        //!< maintaining a valid component list.
+        };
+
+        enum ShareState : int32_t
```

Comment:
Yes, negative values indicate the spawnable is read-only shared, which can be for multiple visitors at the same time. As such this can become larger that -127.